core/dbt/context/base.py

```diff
@@ -16,7 +16,7 @@
     disallow_secret_env_var,
 )
 from dbt.events.functions import fire_event, get_invocation_id
-from dbt.events.types import MacroEventInfo, MacroEventDebug
+from dbt.events.types import JinjaLogInfo, JinjaLogDebug
```

Comment:
Along these same lines there's also a `GeneralMacroWarning` that is the catchall for `warn` defined in `exceptions.py`.  It's worth renaming that along with these.

Reply:
It doesn't look like it's actually used anywhere. Can we just remove it?

Reply:
It's used within macros ([docs](https://docs.getdbt.com/reference/dbt-jinja-functions/exceptions#warn))

Reply:
So it's exported into the context. So should we call in JinjaLogWarning?

Reply:
Or JinjaLogWarn?

Reply:
I went with JinjaLogWarning.